build.gradle

```diff
@@ -20,6 +20,11 @@ configurations {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml'
+    implementation 'com.fasterxml.woodstox:woodstox-core'
```

Comment:
why do u need this `woodstox`?

Reply:
I was following the documentation where they imported it and I didn't realise it is actually unnecessary. Thank you for notice I removed it